### ISSUE TITLE
sympy.physics.mechanics Kane's method / reaction forces

### ISSUE THREAD

**Issue by Peter230655** [sympy/sympy#25887]
Only from observation of many simulations I believe the following may be true: When I want to calculate the reaction forces on a point P, one needs to give this point virtual speeds and virtual forces. 

- If P **has also** real speeds, the force term contains the virtual forces. They must be set to zero, as they perform no work.
- the kinetic energy contains the virtual speeds, which must be set to zero.

This is no big deal for the user, once he know its.
Maybe it would be a slight convenience for the user, if this elimination was done 'automatically'. The virtual speeds are 'known' to the program, one must declare them when setting up Kane's equations. No idea about the virtual forces, one does not have to declare them.
(Unfortunately, my Python knowledge is insufficient to tackle such a thing)

**Comment by moorepants** [comment]
Yes, I think we have an open issue about his already. I agree that it is confusing that aux speeds are not eliminated automatically. 

Here is the issue #13635

**Comment by Peter230655** [comment]
I did not know about issue #13635, this was before I started programming.
So, should I close my issue, since there is another one with the same topic?
